recipes/systemd/systemd.inc

```diff
@@ -0,0 +1,52 @@
+SUMMARY = "A System and service manager"
+HOMEPAGE = "http://www.freedesktop.org/wiki/Software/systemd"
+
+LICENSE = "GPL-2.0 & LGPL-2.1"
+
+inherit autotools-autoreconf pkgconfig
+
+SRC_URI = "git://github.com/systemd/systemd.git;protocol=git;tag=v${PV}"
+
+S = "${SRCDIR}/systemd"
+
+EXTRA_OECONF = " \
+                --enable-split-usr \
+                --without-python \
+                --with-sysvrcnd-path=${sysconfdir} \
+                --with-firmware-path=/lib/firmware \
+		--disable-manpages \
+		--enable-dbus \
+		--enable-resolved \
+		--enable-networkd \
+		--enable-hostnamed \
+		--enable-timedated \
+		--enable-timesyncd \
+		--enable-kdbus \
+		--enable-logind \
+		--enable-sysusers \
+		--enable-bzip2 --enable-lz4 --enable-zlib \
+                "
+#--enable-xz
+
+DEPENDS += "libpthread libcap util-linux-libmount libdl libm librt libresolv libcrypt native:coreutils-ln \
+		dbus-libdbus-1 zlib bzip2-libbz2 lz4-liblz4"
```

Comment:
Please depend on the proper library provides like libdbus-1, libz, libbzip2, liblz4 instead of concrete package names.